.claude/agents/nvim-diffview-specialist.md

```
@@ -0,0 +1,286 @@
+        { "n", "<down>", "<cmd>DiffviewNextEntry<cr>", { desc = "Next entry" } },
+        { "n", "k", "<cmd>DiffviewPrevEntry<cr>", { desc = "Previous entry" } },
+        { "n", "<up>", "<cmd>DiffviewPrevEntry<cr>", { desc = "Previous entry" } },
+        { "n", "<cr>", "<cmd>DiffviewSelectEntry<cr>", { desc = "Select entry" } },
+        { "n", "o", "<cmd>DiffviewSelectEntry<cr>", { desc = "Select entry" } },
+        { "n", "-", "<cmd>DiffviewToggleStage<cr>", { desc = "Stage/unstage" } },
+        { "n", "s", "<cmd>DiffviewToggleStage<cr>", { desc = "Stage/unstage" } },
+        { "n", "S", "<cmd>DiffviewStageAll<cr>", { desc = "Stage all" } },
+        { "n", "U", "<cmd>DiffviewUnstageAll<cr>", { desc = "Unstage all" } },
+        { "n", "X", "<cmd>DiffviewRestoreEntry<cr>", { desc = "Restore entry" } },
+        { "n", "L", "<cmd>DiffviewOpenCommitLog<cr>", { desc = "Open commit log" } },
+        { "n", "g<C-x>", "<cmd>DiffviewCycleLlayout<cr>", { desc = "Cycle layout" } },
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)